Author: YunQiang Su <wzssyqa@gmail.com>
Description: index() is not static, otherwise transfig is liked with
 -L/usr/lib (Closes: #722857)
Bug-Debian: http://bugs.debian.org/722857

--- a/fig2dev/getopt.c
+++ b/fig2dev/getopt.c
@@ -36,7 +36,7 @@
 #define	ENDARGS  "--"
 
 /* this is included because index is not on some UNIX systems */
-static char *
+char *
 index (s, c)
   register  char *s;
   register  int   c;
